compiler/parser/parser.peg

```diff
@@ -82,6 +82,15 @@ FuncDecl
         "expr":expr))
     }
 
+OpDecl
+  = "op" _ name:IdentifierName __ "(" __ params:IdentifierNames __ ")" scope:Scope {
```

Comment:
I'm for making this consistent with FuncDecl by using : instead of => between the parameters and body.
```suggestion
  = "op" _ name:IdentifierName __ "(" __ params:IdentifierNames __ ")" __ ":" __ "(" __ scope:Sequential __ ")" {
```

Reply:
Not opposed to this.